lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -208,7 +208,8 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
               </h2>
 
               <p class="text-gray-700">
-                Deployment groups allow you to deploy notebooks to self-hosted machines with the click of a button.
+                Deployment groups contain the configuration that allows you to deploy Livebook apps
+                to self-hosted machines with the click of a button.
```

Comment:
I am not sure if the "contain the configuration" help. We could use this on top of everything "notebooks contain the code", "secrets contain the name and value", etc. Maybe this:

```suggestion
                Deployment groups allows you to deploy Livebook apps
                to self-hosted machines with the click of a button. Configure
                one or more groups to deploy to different machines.
```

Reply:
@josevalim Ok, I move forward with your suggestion and only removed the last sentence.